apps/slack/lambda/lib/errors/exception.ts

```diff
@@ -1,10 +1,13 @@
+import { ErrorObject } from 'ajv/dist/types';
```

Comment:
I will have to look into avj, I haven't used it before!

Reply:
yes! also new to me!